docs/CHANGELOG.asciidoc

```diff
@@ -1,67 +1,114 @@
+// cannot be empty.
+// TEMPLATE:
 
-=== Breaking Changes
+// [[release-notes-n.n.n]]
+// == {es} n.n.n
 
-=== Deprecations
+//=== Breaking Changes
 
-=== New Features
+//=== Deprecations
 
-Detectors now support rules that allow the user to improve the results by providing some domain specific
-knowledge in the form of rule. ({pull}119[#119])
+//=== New Features
 
-=== Enhancements
+//=== Enhancements
 
-Improve and use periodic boundary condition for seasonal component modeling ({pull}84[#84])
-Improve robustness w.r.t. outliers of detection and initialisation of seasonal components ({pull}90[#90])
-Improve behavior when there are abrupt changes in the seasonal components present in a time series ({pull}91[#91])
-Explicit change point detection and modelling ({pull}92[#92])
-Improve partition analysis memory usage ({pull}97[#97])
-Reduce model memory by storing state for periodicity testing in a compressed format ({pull}100[#100])
-Improve the accuracy of model memory control ({pull}122[#122])
-Improve adaption of the modelling of cyclic components to very localised features ({pull}134[#134])
-Reduce the memory consumed by distribution models ({pull}146[#146])
+//=== Bug Fixes
 
-Forecasting of Machine Learning job time series is now supported for large jobs by temporarily storing
-model state on disk ({pull}89[#89])
+//=== Regressions
 
-Secure the ML processes by preventing system calls such as fork and exec. The Linux implemenation uses
-Seccomp BPF to intercept system calls and is available in kernels since 3.5. On Windows Job Objects prevent
-new processes being created and macOS uses the sandbox functionality ({pull}98[#98])
+//=== Known Issues
+////
 
-Fix a bug causing us to under estimate the memory used by shared pointers and reduce the memory consumed
-by unnecessary reference counting ({pull}108[#108])
+== Elasticsearch version 6.4.0
 
-Reduce model memory by storing state for testing for predictive calendar features in a compressed format
-({pull}127[#127])
+//=== Breaking Changes
 
-=== Bug Fixes
+//=== Deprecations
+
+=== New Features
 
-Age seasonal components in proportion to the fraction of values with which they're updated ({pull}88[#88])
-Persist and restore was missing some of the trend model state ({pull}#99[#99])
-Stop zero variance data generating a log error in the forecast confidence interval calculation ({pull}#107[#107])
-Fix corner case failing to calculate lgamma values and the correspoinding log errors ({pull}#126[#126])
-Influence count per bucket for metric population analyses was wrong and lead to wrong influencer scoring ({pull}#150[#150])
-Fix a possible SIGSEGV for jobs with multivariate by fields enabled which would lead to the job failing ({pull}#170[#170])
+* Detectors now support {stack-ov}/ml-rules.html[custom rules] that enable the user to improve machine learning results by providing some domain-specific knowledge in the form of rule. ({ml-pull}119[#119])
```

Comment:
This change will also probably picked up by the release note script on the Java side - the two notes should be merged when the C++ changelog is merged with the Java one.